lib/image-utils.js

```diff
@@ -15,6 +15,13 @@ function resizeImage(imageUrl, { width, height, query, baseUrl }) {
   if (!query && imageUrl.match(/\.svg$/)) {
     return imageUrl;
   } // if we don't need to transform the image, no need to proxy it.
+
+  if (imageUrl?.startsWith(`${window.location.origin}/api/files/`)) {
+    const url = new URL(imageUrl);
+    url.searchParams.append('thumbnail', '');
+    return url.toString();
+  }
```

Comment:
Maybe we can add some error handling so as not to crash in case the URL is invalid. Also, `startsWith` is not a safe way to check the domain: `https://opencollective.com/api/files/@example.com` will pass and end up redirecting to `example.com`.

To summarize, I think we want something like this:

```es6
try {
  const parsed = new URL(imageUrl);
  if (parsed.hostname === '...' && parsed.pathname === '...') {
    ...
  }
} catch {
  return null;
}
```